kolibri-electron/src/index.js

```diff
@@ -254,4 +259,9 @@ app.on('ready', () => {
 
 app.on('window-all-closed', () => {
   app.quit();
+
+  const pidFile = path.join(KOLIBRI_HOME, 'server.pid');
+  if (fs.existsSync(pidFile)) {
+    fs.rmSync(pidFile);
+  }
```

Comment:
Nit: This is duplicated code that can be in it's own function on top of the file.